### ISSUE TITLE
NuGet Package does not work on Blazor WASM project

### ISSUE THREAD

**Issue by WizX20** [h3x4d3c1m4l/Blazor.LoadingIndicator#8]
I installed the Nuget Package, but I seem to be unable to get it working.

Steps to reproduce:

- Install latest: `dotnet new -i Microsoft.AspNetCore.Blazor.Templates::3.2.0-preview1.20073.1`
- Install the Nuget package
- (Close and re-open solution, rebuild, clean, and all)
- Add package namespace to _imports.razor

![image](https://user-images.githubusercontent.com/1008647/73592071-75f73400-44f6-11ea-88ad-a5c1e00b43b8.png)

**Comment by h3x4d3c1m4l** [comment]
Yep, the NuGet package is out of date. Please add the repo to yours as submodule.
The repo is updated to Blazor 3.2.0-preview1.

Once Blazor is stable, I will update the NuGet package and keep it up to date with the current stable Blazor version. The current NuGet package was just a quick test.

**Comment by WizX20** [comment]
Hi, I would rather not do that. Because of the license this project is using that would cause problems for me.

**Comment by rleath82** [comment]
How would you add this to the initial page load (index.html) to replace the "Loading..."?

**Comment by h3x4d3c1m4l** [comment]
@rleath82 Very good question! As Blazor's runtime is not running at that time yet (because it's still downloading the runtime and assemblies) it's not possible to use this library to achieve that.

I will make an example for something like that, but a very simple form could be made by copying some of the CSS from this library to your index.html (or to a `loading.css`) and adopt the template to your needs. If you need some form of progress (e.g. when your users have a slow internet connection and you want to show app download progress to them), then you could try to intercept the XML HTTP request to track the amount of files that are downloading and the ones that have finished.

**Comment by h3x4d3c1m4l** [comment]
@rleath82 I opened Issue #9 for ^

**Comment by h3x4d3c1m4l** [comment]
@WizX20 I decided to relicense the project to LGPLv3. Please tell me if this works for your project!
